.gitattributes

```diff
@@ -1 +1,4 @@
 *.sol linguist-language=Solidity
+prerelease.lock export-ignore
+commit_hash.txt export-subst
+current_tag.txt export-subst
```

Comment:
Paths here might need the `/` at the beginning if they work like in `.gitignore`.